pom.xml

```diff
@@ -51,6 +51,17 @@
             <artifactId>github</artifactId>
             <version>1.14.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>branch-api</artifactId>
+            <version>0.2-beta-6</version>
```

Comment:
+1 for adding `SCMNavigatrorOwner` in `scm-api-plugin` to be consistent with the current design.

@kzantow This plugin is basically a _provider_ for other entities in a higher level, it should not need to know about _who_ is calling its interfaces.